docs/content/features/run_on_aws.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Run on AWS EC2 Instances"
+draft: false
+description: "Run Infection Monkey on AWS EC2 instances"
+tags: ["aws", "ec2", "run"]
+pre: "<i class='fa-brands fa-aws'></i> "
+---
+
+## Description
+
+The Infection Monkey can run on AWS EC2 instances using the
+[AWS Systems Manager (SSM) Agent](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html).
+Each EC2 instance has an [SSM Agent](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html)
+installed for updating, managing, and configuring the instance.
+It executes requests received from the AWS Systems Manager, and uses the Amazon
+Message Delivery Service to report back status and execution details.
```

Comment:
This context is fantastic. Should we add links for AWS Systems Manager and AMDS? Or are they not important/